### ISSUE TITLE
Nothing significant just spelling of a district

### ISSUE THREAD

**Issue by GHSScienceMACC** [triplea-game/triplea#10018]
The district named Parisburg. I'm assuming the town in Virginia. It is spelled Pearisburg. Just figured you might want to know.

**Comment by bacrossland** [comment]
@beelee1 the corrected spelling for this is in the above map PR. If you could review it, I would appreciate it. Once the PR is merged, this issue can be closed.

**Comment by Cernelius** [comment]
Does everyone realize that changing a zone name in a map breaks backward compatibility with saved games? Meaning that everyone which has saved games of the map and updates the map will be unable to continue his/her saved games. This is especially problematic in the moment I believe TripleA fails to warn the player of this fact.

I tried to be informative here (but was refused):
https://github.com/triplea-game/triplea/pull/9882

**Comment by Cernelius** [comment]
Also, be sure to check the spelling on an original Civil War era atlas or map, or an alternative reliable source. We can assume the names are supposed to be the ones of the period and names (if only their spelling) sometimes change overtime.

**Comment by bacrossland** [comment]
> Does everyone realize that changing a zone name in a map breaks backward compatibility with saved games? Meaning that everyone which has saved games of the map and updates the map will be unable to continue his/her saved games. This is especially problematic in the moment I believe TripleA fails to warn the player of this fact.
> 
> I tried to be informative here (but was refused): #9882

I believe everyone that is a developer and a longer time player is aware of that. It's one of the reasons a lot of players don't update the map or upgrade the software until after they have finished the game they currently have saved. I read the entire discussion between you and @DanVanAtta in which he makes that exact point when talking about over wordy and scarily worded warns. However, never updating maps and software because it has the potential to cause a saved game not to function is unreasonable. TripleA software and it's maps must move forward and by nature that will cause older saved games to be not compatible. This is precisely why he moved forward with the change.

**Comment by Cernelius** [comment]
Fair enough. I was mostly wondering if changing Parisburg to Pearisburg was an improvement worth breaking compatibility with ongoing games, but I do support naming correctess (if it is correct), so I don't have a strong opinion either way.

**Comment by bacrossland** [comment]
> Also, be sure to check the spelling on an original Civil War era atlas or map, or an alternative reliable source. We can assume the names are supposed to be the ones of the period and names (if only their spelling) sometimes change overtime.

Pearisburg was founded in 1808 when Giles Country was established and named after the landowner George Pearis, a local, who donated a 50-acre tract to be used for a town that would grow up around the county courthouse. Maps of Virginia at the time incorrectly spelled Pearisburg as Parisburg because the phonetic pronunciation of the name. For more information contact the Giles County Historical Society located in Pearisburg, VA (https://www.countyoffice.org/giles-county-historical-society-pearisburg-va-f50/)

Using names from historical maps can be a reference but should not be the end to line when it comes to what a location on a modern wargame maps should be labeled. Historical maps do have incorrect spellings of names and places that are carried forward for years because the next map is based off the last one. Unless you are using an actual image of the historical map for your game or using the name from the correct time period the game is based (Istanbul vs Constantinople), you should not use the incorrect spelling of a location that has not changed the spelling of its name for its 214 year existence regardless of where you got that incorrect spelling.

**Comment by Cernelius** [comment]
> > Also, be sure to check the spelling on an original Civil War era atlas or map, or an alternative reliable source. We can assume the names are supposed to be the ones of the period and names (if only their spelling) sometimes change overtime.
> 
> Pearisburg was founded in 1808 when Giles Country was established and named after the landowner George Pearis, a local, who donated a 50-acre tract to be used for a town that would grow up around the county courthouse. Maps of Virginia at the time incorrectly spelled Pearisburg as Parisburg because the phonetic pronunciation of the name. For more information contact the Giles County Historical Society located in Pearisburg, VA (https://www.countyoffice.org/giles-county-historical-society-pearisburg-va-f50/)
> 
> Using names from historical maps can be a reference but should not be the end to line when it comes to what a location on a modern wargame maps should be labeled. Historical maps do have incorrect spellings of names and places that are carried forward for years because the next map is based off the last one. Unless you are using an actual image of the historical map for your game or using the name from the correct time period the game is based (Istanbul vs Constantinople), you should not use the incorrect spelling of a location that has not changed the spelling of its name for its 214 year existence regardless of where you got that incorrect spelling.

Actually, I would be fine using the incorrect spelling if that was an used spelling for the time. When there are multiple possible spellings and some of them went out of use, I actually tend to prefer using the archaic ones to give the map an antique ring. The notion of a spelling being "incorrect" is itself relative because this is how language mostly evolves (but not in this case).

I checked nothing myself, but I was expecting exactly something like this. I would not be surprised if the original mapmaker specifically chose Parisburg over Pearisburg. If not, he was likely simply looking at a period map having the "wrong" spelling.

**Comment by beelee1** [comment]
@bacrossland yea i'm not a developer. Dan just added me so i can merge peoples maps. I think Roi is probably the senior active developer at the moment
